src/main/java/com/robinpowered/sdk/model/Identifier.java

```diff
@@ -72,6 +72,9 @@ public static Interface fromString(String value) {
     private final String value;
     private final DateTime createdAt;
 
+    // Mutable
+    private Integer deviceId;
```

Comment:
Ah, thanks for logging my memory...this makes a lot of sense now. We had it polymorphic on `account_id/device_id`. Aight that clears it up